aiowebostv/webos_client.py

```diff
@@ -202,6 +210,7 @@ async def connect_handler(self, res):
             await asyncio.wait(handler_tasks, return_when=asyncio.FIRST_COMPLETED)
 
         except Exception as ex:  # pylint: disable=broad-except
+            _LOGGER.debug("exception(%s): %s", self.host, repr(ex), exc_info=True)
```

Comment:
Why use `repr`?
```suggestion
            _LOGGER.debug("exception(%s): %s", self.host, ex, exc_info=True)
```

Reply:
I changed it to `_LOGGER.debug("exception(%s): %r", self.host, ex, exc_info=True)` since printing the exception as a string would print an empty string